addons/medical_treatment/functions/fnc_handleBandageOpening.sqf

```diff
@@ -26,6 +26,7 @@ private _className = EGVAR(medical_damage,woundClassNamesComplex) select _classI
 private _reopeningChance = DEFAULT_BANDAGE_REOPENING_CHANCE;
 private _reopeningMinDelay = DEFAULT_BANDAGE_REOPENING_MIN_DELAY;
 private _reopeningMaxDelay = DEFAULT_BANDAGE_REOPENING_MAX_DELAY;
+private _reopeningModifer = GVAR(woundReopenChance);
```

Comment:
Just use the global variable directly on L88.

Reply:
Resolved & tested. Thanks.